.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "third_party/goog"]
 	path = third_party/goog
 	url = https://github.com/tart/closure-library.git
+[submodule "third_party/jaguarjs-jsdoc"]
```

Comment:
We should find another way of including this in our projects. if travis allows bash scripts, we could just clone it. Or we could publish it to npm, and then just do an `npm install`; declaring `jaguarjs-jsdoc` as one of our `devDependencies`